src/core/components/tabs/tabsTrigger/tabsTrigger.tsx

```diff
@@ -0,0 +1,48 @@
+import { TabsTrigger as RadixTabsTrigger, type TabsTriggerProps as RadixTabsTriggerProps } from '@radix-ui/react-tabs';
+import classNames from 'classnames';
+import { useContext } from 'react';
+import { Icon, type IconType } from '../../icon';
+import { TabsContext } from '../tabsRoot/tabsRoot';
+export interface ITabsTriggerProps extends RadixTabsTriggerProps {
+    /**
+     * The label of the tab.
+     */
+    label: string;
+    /**
+     * Value of the tab is meant to pair with the Tabs.Content value prop.
+     */
+    value: string;
```

Comment:
Should value be required if tabs can be used without content?

Reply:
It's required by Radix by default so I would say yes. I'm only including here to extend documentation as it's an essential implementation detail for the broader component module.